plugins/statistics.koplugin/main.lua

```diff
@@ -428,6 +428,14 @@ Please wait…
             end
         end
     end
+    if not self.settings.fixed_null_idbook then
```

Comment:
You could probably just increment the db version thing (I don't recall the exact name, and am on mobile right now, sorry) instead of adding a setting key that'll stay around.

Reply:
It's just that the current migration function will backup the db, which might be 2GB! And I'm reluctant to give that workflow a special case :(

Reply:
Ah, good point, I'd forgotten about that.

Perhaps we can just move that whole bit of logic to a onetimemigration block instead of the plug-in itself, then?

Reply:
Done. Didn't really know about the onetime_migration choice.